src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -107,6 +107,92 @@ RppStatus rppt_gridmask_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** glitch ********************/
+
+RppStatus rppt_glitch_host(RppPtr_t srcPtr,
```

Comment:
Please add glitch as the last one i.e after rppt_gaussian_noise_host and before the GPU API's